DECISION-MAKING.md

```diff
@@ -1,143 +1,154 @@
 
-- Owner (accountable, decision maker): @akrug
+- IT Capability Owner: @jeffbryner
+- Owner: @akrug
 - Peers: @gdestuynder
-- Operator (responsible, developer): EIS
+- Operator: EIS
 
 Repository:
 - https://github.com/mozilla-iam/sso-dashboard
 
-## SSO Dashboard configuration and access control database
+## SSO Dashboard Configuration and Access Control Database
 
 This is the configuration and `apps.yml`.
 
-- Owner (accountable, decision maker): @jdow
+- IT Capability Owner: @jeffbryner
+- Owner: @jdow
 - Peers: @akrug @gdestuynder
-- Operator (responsible, developer): EIS
+- Operator: EIS
 
 Repository:
 - https://github.com/mozilla-iam/sso-dashboard-configuration
 
 ## GSuite Community Driver
 
-This is the team drive integration.
+This is the G Suite Team Drive integration.
 
-- Owner (accountable, decision maker): @akrug
+- IT Capability Owner: @jeffbryner
+- Owner: @akrug
 - Peers: @gdestuynder
-- Operator (responsible, developer): EIS
+- Operator: EIS
 
 Repository:
 - https://github.com/mozilla-iam/gsuite-community-drive-driver
 
 ## Slack Driver
 
-This is the Slack deprovisioning driver.
+This is the driver for Slack user deprovisioning.
 
-- Owner (accountable, decision maker): @gdestuynder
+- IT Capability Owner: @jeffbryner
+- Owner: @gdestuynder
 - Peers: @akrug
-- Operator (responsible, developer): EIS
+- Operator: EIS
 
 Repository:
 - https://github.com/mozilla-iam/slack-driver/
 
 ## New Login Experience (NLX)
 
-This is the login window for Mozilla IAM.
+This is the login window for _Mozilla IAM_.
 
-- Owner (accountable, decision maker): @hidde
-- Peers: @akrug @gdestuynder
-- Operator (responsible, developer): EIS
+- IT Capability Owner: @jeffbryner
+- Owner: @hidde
+- Peers: @akrug @gdestuynder <can we add a full staff member from ParSys here?>
+- Operator: EIS
 
 Repository:
 - https://github.com/mozilla-iam/auth0-custom-lock
 
-## Mozillians.org profile editor
+## Mozillians.org Profile and Group Management
 
-This is https://www.mozillians.org which manages profiles, groups, etc.
+This is the place to edit _Mozilla IAM_ profiles, manage groups, search & discover people. It is available at https://mozillians.org.
 
-- Owner (accountable, decision maker): @hmitsch
-- Peers: @johngian @akatsoulas @fiji-flo
-- Operator (responsible, developer): Parsys
+- Owner: @hmitsch
+- Peers: @johngian @akatsoulas @fiji-flo <suggesting to add a InfoSec person?>
+- Operator: ParSys
 
 Repository:
 - https://github.com/mozilla/mozillians
 
 ## Phonebook
 
-This is https://phonebook.mozilla.org
+This is the Staff-only accessible system to access Mozilla's org chart and Staff profiles, available at https://phonebook.mozilla.org.
 
-- Owner (accountable, decision maker): @akrug
+- IT Capability Owner: @jeffbryner
+- Owner: @akrug
 - Peers: @atoll
-- Operator (responsible, developer): IT
+- Operator: IT (MOC?)
 
 Repository:
 - https://github.com/mozilla/phonebook
+
+## IAM Infrastructure
```

Comment:
can we define what IAM Infrastructure is?

Reply:
Yes, I was hoping you could help me with some text here. :-)